### PR TITLE
spec 'erase' param on /account/deactivate

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/matrix-org/matrix-doc/issues/1288

### CODE REVIEW

File: api/client-server/registration.yaml [hunk 1]
```diff
@@ -348,6 +348,11 @@ paths:
                 description: |-
                   Additional authentication information for the user-interactive authentication API.
                 "$ref": "definitions/auth_data.yaml"
+              erase:
+                description: |-
+                  Indicates whether the account should be erased. Defaults to `false`.
```
Comment (turt2live) [inline api/client-server/registration.yaml]: What is the server expected to erase?

Comment (t3chguy) [reply api/client-server/registration.yaml]: I copied the doc comment from js sdk :(

**Comment by maxidor** [comment]
Spec proposal?

**Comment by t3chguy** [comment]
the issue is marked as `spec-omission` so clearly its already accepted into the spec, just not documented.

**Comment by non-Jedi** [comment]
I disagree that this is a spec omission. All of the stuff added for GDPR needs to go through the formal proposal process before it's added to the spec. It's not like this is stuff that's been implemented in riot-web/synapse for forever and we're just documenting it now. This is behavior that's been added either after the proposal process was implemented or close to concurrent with it.

**Comment by t3chguy** [comment]
Then the triaging needs to change.

**Comment by turt2live** [comment]
There's been quite a few mentions that `spec-omission` doesn't always mean it can go straight to PR. I believe this is one of those cases where an overarching GDPR proposal is needed.

I'll add this to the bucket of GDPR things though

**Comment by richvdh** [comment]
ok let's close this for now pending a GDPR spec.

**Pull request closed without merging**